client/me/billing-history/transactions-header.jsx

```diff
@@ -33,7 +33,7 @@ class TransactionsHeader extends React.Component {
 		event.preventDefault();
 	};
 
-	componentWillMount() {
+	UNSAFE_componentWillMount() {
```

Comment:
This one seems unrelated? What's the scoop?

Reply:
Removed it as unrelated change. The `closest` call in this file didn't need a change.